Makefile

```diff
@@ -0,0 +1,16 @@
+SPARK_ON_K8S_OPERATOR_REPO_NAME ?= mesosphere
+SPARK_ON_K8S_OPERATOR_IMAGE_NAME ?= spark-on-k8s-operator
+SPARK_ON_K8S_OPERATOR_FULL_IMAGE_NAME ?= $(SPARK_ON_K8S_OPERATOR_REPO_NAME)/$(SPARK_ON_K8S_OPERATOR_IMAGE_NAME)
+SPARK_ON_K8S_DOCKER_FILE_PATH ?= shared/spark-on-k8s-operator
+SPARK_DOCKER_IMAGE_NAME ?= "gcr.io/spark-operator/spark:v2.4.0"
+
+.PHONY: operator-docker-build
+operator-docker-build:
```

Comment:
how about renaming this and the target below to just `docker-build` and `docker-push`?

Reply:
renamed as suggested